bravado/testing/response_mocks.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+from bravado.exception import BravadoTimeoutError
+from bravado.http_future import FALLBACK_EXCEPTIONS
+from bravado.response import BravadoResponseMetadata
+
+
+class BravadoResponseMock(object):
+    def __init__(self, result):
+        self._result = result
+
+    def __call__(self, timeout=None, fallback_result=None, exceptions_to_catch=FALLBACK_EXCEPTIONS):
+        return self
+
+    @property
+    def result(self):
+        return self._result
+
+    @property
+    def metadata(self):
+        return BravadoResponseMetadata(
+            incoming_response='incoming response',
+            swagger_result=self._result,
+            start_time=1528733800,
+            request_end_time=1528733801,
+            handled_exception_info=None,
+            request_config=None,
+        )
+
+
+class DegradedBravadoResponseMock(object):
```

Comment:
Add a docstring to explain how this class is supposed to be used, potentially with an example.

Reply:
I went a bit further and wrote documentation. :)